spec/models/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+require "spec_helper"
```

Comment:
Why do we need to require these?

Reply:
we do not need the spec_helper here because it is already required in rails_helper. but rails_helper is needed because shoulda matchers configuration is used in the user_spec so it needs to know where to look for the shoulda matchers

Reply:
spec_helper removed, rails helper stays

Reply:
added rails helper in rspec no need of rails helper anymore. removing it too.